src/cursor/cur_bulk.c

```diff
@@ -23,7 +23,7 @@ __bulk_col_keycmp_err(WT_CURSOR_BULK *cbulk)
 
     WT_RET_MSG(session, EINVAL,
       "bulk-load presented with out-of-order keys: %" PRIu64
-      " is less than previously inserted key %" PRIu64,
+      " is less or equal than previously inserted key %" PRIu64,
```

Comment:
is less than or equal to

Reply:
Thanks see 1435fbed87a92092bf726c0d5a5b0948bcd93f4a